### PR TITLE
Clean up lint errors and add documentation to proto files.

### PR DESCRIPTION
**Pull request by mansu**
Clean up lint errors and add documentation to proto files.

### CODE REVIEW

**Comment by vthacker** [comment]
+0 to this. How will we catch linter errors in the future? 

I think if we do this let's make sure the linter rule that you used is added as a pre-commit step

**Review by bryanlb — CHANGES_REQUESTED** [review]
If we're doing this I think we should apply the cleanup to all files instead of just one. I'd also like to see this made part of the build process validation if we're going to introduce this.

File: kaldb/src/main/proto/kaldb_configs.proto [hunk 1]
```diff
@@ -4,127 +4,157 @@ package slack.proto.kaldb;
...
+  int64 stale_duration_secs= 5;
+  // Name of the data transformation pipeline to use when ingesting the data.
+  string data_transformer = 6;
+  // Folder where data is persisted locally on disk.
+  string data_directory = 7;
+  // Indexer server config.
+  ServerConfig server_config = 8;
+  // MaxOffsetDelay is the maximum delay between the current Kafka
+  // head and the current position the indexer can catch up on.
+  int64 max_offset_delay = 9;
```
[the remark this answers is not in the document]
Comment (mansu) [reply kaldb/src/main/proto/kaldb_configs.proto]: Thought I deleted it. Dropped it now.

**Comment by mansu** [comment]
I agree with doing a better job at this. Adding the protolint now is quite a bit of yak shaving so let's tackle it at a different time. Opened an issue for it https://github.com/slackhq/kaldb/issues/225.

**Review by vthacker — DISMISSED** [review]

**Review by bryanlb — APPROVED** [review]
:shipit:

**Pull request merged**